porter/v1/addons.proto

```diff
@@ -88,9 +90,11 @@ message Addon {
     Deepgram deepgram = 12;
     // N8n is the configuration for the n8n addon
     N8n n8n = 14;
+    // mistral7b is the configuration for the mistral 7b model addon
+    Mistral7B mistral7b = 15;
   }
   // helm_overrides is stringified json of raw Helm overrides to use for the addon. These will override any values generated by the contract.
-  HelmOverrides helm_overrides = 13;
+  HelmOverrides helm_overrides = 20;
```

Comment:
why change this?

Reply:
I was going for readability. It reads better when the numbers are ascending. But it is not worth making this a breaking change. For the future, it is better to just spread these numbers out if we expect to make changes like this one.